virttest/env_process.py

```diff
@@ -62,6 +62,14 @@
 _setup_manager = test_setup.SetupManager()
 
 
+#: Hooks to use for own customization stages of the virtual machines
+#: with params and env as supplied arguments
+preprocess_vm_off_hook = None
+preprocess_vm_on_hook = None
```

Comment:
Is that possible to have hook chains? Just a rough idea, request for your comments.

Reply:
I would say the current hoots are most useful as entry points for your own code, once inside your own custom function you can can just implement a chain of calls. Correct me if I misunderstood what you mean of course.